src/docs/development/ui/layout/index.md

```diff
@@ -72,170 +72,158 @@ the children should occupy.
 
 ## Lay out a widget
 
-{{site.alert.secondary}}
-  <h4 class="no_toc">What's the point?</h4>
```

Comment:
@sfshaza2 - I dropped the "What's the point" alert for this section. Most points didn't seem that helpful, except for one -- concerning `Scaffold` -- which I moved into the main text because it wasn't even discussed there. WDYT?

Reply:
OK